profiles/kentik_snmp/raritan/raritan-pdu.yml

```diff
@@ -0,0 +1,75 @@
+#https://mibs.observium.org/mib/PDU2-MIB/
+---
+extends:
+  - system-mib.yml
+
+provider: kentik-pdu
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.6    # Raritan PX2 PDU2
+
+metrics:
+  - MIB: RARITAN-PX2-PDU2-MIB
+table:
+    OID: 1.3.6.1.4.1.13742.6.3.5.4.1
+    name: outletSensorConfigurationTable
+  symbols:
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.1
+      name: outletSensorTolerance
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.2
+      name: outletSensorMaximum
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.3
+      name: outletSensorMinimum
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.4
+      name: outletSensorHysteresis
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.6
+      name: outletSensorUnits
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.7
+      name: outletSensorDecimalDigits
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.8
+      name: outletSensorAccuracy
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.9
+      name: outletSensorResolution
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.14
+      name: outletSensorStateChangeDelay
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.21
+      name: outletSensorLowerCriticalThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.22
+      name: outletSensorLowerWarningThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.23
+      name: outletSensorUpperCriticalThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.24
+      name: outletSensorUpperWarningThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.25
+      name: outletSensorEnabledThresholds
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.26
+      name: outletSensorSignedMaximum
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.27
+      name: outletSensorSignedMinimum
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.28
+      name: outletSensorSignedLowerCriticalThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.29
+      name: outletSensorSignedLowerWarningThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.30
+      name: outletSensorSignedUpperCriticalThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.31
+      name: outletSensorSignedUpperWarningThreshold
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.4
+      name: outletSensorLogAvailable
```

Comment:
@al3kh - need to fix this indentation please

also; for a lot of these, consider if they're volatile (change often) or not. I'd suspect that a number of these Threshold and Min/Max OIDs are fairly static and can be moved into the `metric_tags` section (for this table, not the global section at the bottom).

This helps a lot with your data ingest because it "compresses" the values as decorations on the actual metrics you leave under `symbols` as opposed to every single item being its own dimensional metric. Tags are also refreshed every 30 minutes by `ktranslate`, which is a lot more conservative than a standard metric that you collect every 5 minutes or less.